packages/blade/src/components/IconButton/__tests__/IconButton.native.test.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+import { fireEvent } from '@testing-library/react-native';
+
+import IconButton from '../';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+import { CloseIcon } from '~components/Icons';
+
+describe('<IconButton />', () => {
+  it('should render', () => {
+    const noop = () => {};
+    const { toJSON } = renderWithTheme(
+      <IconButton accessibilityLabel="Close" icon={CloseIcon} onClick={noop} />,
+    );
+
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should handle callback', () => {
```

Comment:
Do you mean should handle onClick? Callback would be the function passed as 1st argument to `onClick`. We're not asserting that here anywhere? Refer Button/Link's tests for callback vs onclick tests

Reply:
Resolved in https://github.com/razorpay/blade/pull/613/commits/a7c992a1205f00da61795e47039c46eb4457f64a